composer.json

```diff
@@ -16,7 +16,7 @@
         "doctrine/orm": "^2.11",
         "easycorp/easyadmin-bundle": "^3.5",
         "itk-dev/database-bundle": "^1.3",
-        "itk-dev/getorganized-api-client-php": "dev-develop",
+        "itk-dev/getorganized-api-client-php": "dev-feature/custom-http-client",
```

Comment:
https://github.com/itk-dev/getorganized-api-client-php/pull/12 Has been merged, so this would also need to be reverted now, right?